test_communication/test/test_messages_c.cpp

```diff
@@ -129,25 +129,25 @@ class CLASSNAME (TestMessagesFixture, RMW_IMPLEMENTATION) : public ::testing::Te
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string_safe();
     EXPECT_EQ(0, strcmp(rcl_subscription_get_topic_name(&subscription), topic_name));
 
-    rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
+    rcl_wait_set_t wait_set_connection = rcl_get_zero_initialized_wait_set();
```

Comment:
You're right it's better to just wrap it in a local scope, which is what @wjwwood suggested initially I think.

Fixed in https://github.com/ros2/system_tests/pull/189/commits/ffce07c9c26559f538a4cda614e2d9cb2cd389a0